server/views/pages/dashboard.njk

```diff
@@ -20,24 +20,26 @@
     }) }}
   </div>
   <div class="horizontal-form__wrapper govuk-!-padding-3 govuk-!-margin-bottom-5">
-    <h2 class="govuk-heading-m">Filter by</h2>
+    <h2 class="govuk-heading-l">Incident reports</h2>
```

Comment:
the first heading should still be an h1 (even if it’s supposed to look small) for proper doc structure. e.g. it helps screen readers.